File: 백준/Bronze/14038. Tournament Selection/Tournament Selection.c
#include <stdio.h>

int main()
{
    char s;
    int i, count = 0;

    for (i = 0; i < 6; i++) {
        scanf("%c\n", &s);
        if (s == 'W')
            count++;
    }

    if (count == 0)
        printf("-1");
    else if (count >= 5)
        printf("1");
    else if (count >= 3)
        printf("2");
    else
        printf("3");

    return 0;
}

File: 백준/Bronze/14038. Tournament Selection/Tournament Selection.py
count = 0
for i in range(6):
    s = input()
    if s == 'W':
        count += 1
if count == 0:
    print(-1)
elif count >= 5:
    print(1)
elif count >= 3:
    print(2)
else:
    print(3)